core/controlplane/config/config.go

```diff
@@ -63,6 +63,9 @@ func NewDefaultCluster() *Cluster {
 			Disk:       "xvdb",
 			Filesystem: "xfs",
 		},
+		ClusterTLSBootstrap: ClusterTLSBootstrap{
+			Enabled: false,
+		},
```

Comment:
Very nit picky but moving this block to the line 61 would be better for a consistent ordering between this func and the definition of the Experimental struct.